dutchx_scripts/daoCreateDutchX.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  Address,
+  Auction4ReputationFactory,
+  ConfigService,
+  DAO,
+  ExternalLocking4ReputationFactory,
+  InitializeArcJs,
+  LockingEth4ReputationFactory,
+  LockingToken4ReputationFactory,
+  Utils,
+  Web3
+} from "@daostack/arc.js";
+
+import { run as contractNew } from "../scripts/contractNew";
+import { run as daoCreate } from "../scripts/daoCreate";
+import { run as tokenMint } from "../scripts/tokenMint";
+
+export const run = async (web3: Web3, networkName: string): Promise<void> => {
+
+// tslint:disable: max-line-length
+
+  await InitializeArcJs({
+    filter: {},
+  });
+
+  ConfigService.set("estimateGas", true);
+
+  const daoSchema = require(`../../dutchx_scripts/specsNewDaos/dutchX.${networkName.toLowerCase()}.json`);
+
+  const lockingEth4Reputation =
+    await LockingEth4ReputationFactory.at((await contractNew(web3, networkName, { name: "LockingEth4Reputation" }, "max", "50000000000") as Lock4ReputationContract).address);
+  const lockingToken4Reputation =
+    await LockingToken4ReputationFactory.at((await contractNew(web3, networkName, { name: "LockingToken4Reputation" }, "max", "50000000000") as Lock4ReputationContract).address);
+  const externalLocking4Reputation =
+    await ExternalLocking4ReputationFactory.at((await contractNew(web3, networkName, { name: "ExternalLocking4Reputation" }, "max", "50000000000") as Lock4ReputationContract).address);
+  const auction4Reputation =
+    await Auction4ReputationFactory.at((await contractNew(web3, networkName, { name: "Auction4Reputation" }, "max", "50000000000") as Lock4ReputationContract).address);
+
+  let externalTokenLockerAddress: Address;
+  let priceOracleInterfaceAddress: Address;
+  let priceOracleInterfaceMock: any;
+  let gnoTokenAddress: Address;
+  let externalTokenLockerMock: { lock: (amount: string, options: { from: Address }) => Promise<void>, address: Address };
```

Comment:
this not suppose to be on master.
it should be conditioned with a param from config file

Reply:
https://github.com/daostack/dx-deployment-script/pull/1/commits/a376b5a0bc59d8546654d35d67903874741d8db3